src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -178,7 +178,7 @@ In this message, the phrase "should not be reached" means that this error messag
     <comment>The abbreviation "JDK" should not be translated.</comment>
   </data>
   <data name="XA0031" xml:space="preserve">
-    <value>Java SDK {0} or above is required when using {1}.</value>
+    <value>Java SDK {0} or above is required when using {1}.  Note: the Android Designer is incompatible with JDK-11: https://aka.ms/vs2019-and-jdk-11</value>
```

Comment:
The message before says `Java SDK 11.0`, should we use the same term in both sentences?
```suggestion
    <value>Java SDK {0} or above is required when using {1}.  Note: the Android Designer is incompatible with Java SDK 11.0: https://aka.ms/vs2019-and-jdk-11</value>
```